.github/workflows/create-gui-pr.yml

```diff
@@ -0,0 +1,92 @@
+name: create-gui-pr
+
+# Ensures that we only run one workflow per branch at a time.
+# Already running workflows will be cancelled.
+concurrency:
+  cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+# TODO: Testing. Switch to branch-specific version.
+on: push
+# on:
+#   push:
+#     branches:
+#       master
+
+env:
+  HOST_REPOSITORY: kumahq/kuma
+  HOST_DIST_DIRECTORY: "app/kuma-ui/pkg/resources/data"
+
+jobs:
+  # Creates a pull request in the main application to update its GUI.
+  create-gui-pr:
+    name: Create “update GUI” PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate GitHub App token
+        # https://github.com/tibdex/github-app-token
+        # Note: Specifically references https://github.com/tibdex/github-app-token/commit/f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06 (v1.6.0) to ensure that this exact version of the workflow action is used. This is a security precaution as automatically opting into newer versions of the action could leak or get the app token stolen.
+        uses: tibdex/github-app-token@f717b5ecd4534d3c4df4ce9b5c1c2214f0f7cd06
+        id: generate-token
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
```

Comment:
I suspect you'll need to [set `repository`](https://github.com/tibdex/github-app-token#example-workflow) to get a token with permissions for another repo

Reply:
That sounds reasonable, but it does seem to work already: https://github.com/kumahq/kuma/pull/4925

Reply:
Yeah it does